.github/workflows/deploy_pages.yaml

```diff
@@ -26,28 +30,12 @@ jobs:
       - uses: dart-lang/setup-dart@e630b99d28a3b71860378cafdc2a067c71107f94
       
       # Build the markdown playground.
-      - name: "markdown: pub get"
-        run: dart pub get
-        working-directory: pkgs/markdown
-      - name: "markdown: build playground"
-        run: >
-          dart compile js -o example/app.dart.js
-          --minify --no-frequency-based-minification --no-source-maps
-          example/app.dart
-        working-directory: pkgs/markdown/example
-
-      # Create the _site directory.
```

Comment:
We'll still need this bit, in order to handle deploying more than just pkgs/markdown,